### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by cancelei**
Hello Reviewer,

Highlights of this part of the Project:

- Add any missing JSX elements to your project (without adding extra functionality)
- Your final design should match Zeplin's design (this includes all elements + the same styling)
- You may use the CSS rules detailed in Zeplin to style the website.

Happy review.

### CODE REVIEW

File: src/styles/BookList.module.css [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+.bookButtons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.divider {
+  border: 1px solid #e0e0e0;
+  margin: 0 5%;
+}
```
Comment (andersonlebon) [inline src/styles/BookList.module.css:36]: - [ ] Kindly make sure your Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) as the requirements say, you should follow (the layout, colors, padding, margin...). check here the difference


Your design | The template 
-----|-----
image | image
image | image

File: src/styles/BookList.module.css [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+/* // bookButtons should be just text buttons in color blue. */
+.bookButton {
+  color: #0290ff;
+  font-family: Montserrat, sans-serif;
+
+  /* font-size: 0.813rem; */
```
Comment (andersonlebon) [inline src/styles/BookList.module.css]: ## [OPTIONAL]

- [ ] Kindly remove commented code as they will make your code clean and clear

File: src/styles/CreateBook.module.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  background-color: #fafafa;
+}
+
+.createBookForm input,
+.createBookForm select {
+  width: 100%;
+  padding: 10px;
+  font-size: 1em;
+
+  /* border: 1px solid #ddd; */
```
Comment (andersonlebon) [inline src/styles/CreateBook.module.css]: ## [OPTIONAL]

- [ ] Kindly remove commented code as they will make your code clean and clear

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @cancelei 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @cancelei 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - Changes requested have been implemented ✅ 
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**